payments/models.py

```diff
@@ -720,7 +721,13 @@ def set_status(self, new_status: OrderStatus, comment: str = None) -> None:
             return
 
         valid_status_changes = {
-            OrderStatus.WAITING: (
+            OrderStatus.DRAFTED: (
+                OrderStatus.OFFERED,
+                OrderStatus.PAID_MANUALLY,
+                OrderStatus.ERROR,
```

Comment:
Is the transition DRAFTED -> ERROR ever used? If it's not needed it would be nice to remove, then we'd not need to worry about orders in ERROR state that don't have a due_date

Reply:
True. I left it here in case it would be needed at some point but now that I think about it, it's best to be left out.

Reply:
Now that I think about it, this is probably needed in the case when the email is missing for example. So:
`leases/services/invoice.py:192`

Reply:
I think I will change the logic so that when sending the invoices in `leases/services/invoice.py:286` the order would be in DRAFTED state and it would be transitioned to OFFERED in `payments.utils.approve_order`

Reply:
The lease is already being set to offered
```python
# leases/services/invoice.py:345
if order.lease and order.order_type == OrderType.LEASE_ORDER:
    # Update lease status
    order.lease.status = LeaseStatus.OFFERED
    order.lease.save()
```

Reply:
But don't we want to set the Order as OFFERED there as well?

Reply:
AHHHH, I was looking for the place where we were setting the order to offered, but then realised that this is what this PR is for 🤦 

Yes, let's do it here 👍